src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/quota.dashboard.json

```diff
@@ -197,7 +197,7 @@
             "dimension": "none",
             "metricName": "select",
             "rawQuery": true,
-            "rawQueryString": "customEvents\r\n| where $__timeFilter(timestamp)\n| where name == 'AzureResourceUsage'\r\n| project\r\n    timestamp,\r\n    resource = tostring(customDimensions.name),\r\n    location = tostring(customDimensions.location),\r\n    current = toreal(customMeasurements.current),\r\n    ['limit'] = toreal(customMeasurements.['limit'])\r\n| where resource != 'NetworkWatchers'\n| where location == 'westus'\n| extend ['percent of limit'] = iff(['limit'] == 0, 0.0, current/['limit'] * 100)\n| summarize ['percent of limit'] = max(['percent of limit']) by bin(timestamp, $__interval), resource\r\n| order by timestamp asc, resource",
+            "rawQueryString": "customEvents\r\n| where $__timeFilter(timestamp)\n| where name == 'AzureResourceUsage'\r\n| project\r\n    timestamp,\r\n    resource = tostring(customDimensions.name),\r\n    location = tostring(customDimensions.location),\r\n    current = toreal(customMeasurements.current),\r\n    ['limit'] = toreal(customMeasurements.['limit']),\n    subscription = tostring(customDimensions.subscription)\n| where resource != 'NetworkWatchers'\n| where subscription == \"68672ab8-de0c-40f1-8d1b-ffb20bd62c0f\"\n| where location == 'westus'\n| extend ['percent of limit'] = iff(['limit'] == 0, 0.0, current/['limit'] * 100)\n| summarize ['percent of limit'] = max(['percent of limit']) by bin(timestamp, $__interval), resource\r\n| order by timestamp asc, resource",
```

Comment:
This limits the query results to just HelixProd, always. This won't do the right thing in the staging instance of Grafana (no data will be shown, because no HelixProd data is available). You probably want to filter for both subscription GUIDs.